src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipe.groovy

```diff
@@ -159,14 +159,15 @@ class ConanHostedRecipe
     createRoute(builder, uploadConaninfo(), CONAN_INFO, hostedHandler.uploadConanInfo)
     createRoute(builder, uploadConanPackageZip(), CONAN_PACKAGE, hostedHandler.uploadConanPackage)
     createRoute(builder, uploadConanSources(), AssetKind.CONAN_SOURCES, hostedHandler.uploadConanSources)
-    createRoute(builder, uploadConanExportZip(), CONAN_PACKAGE, hostedHandler.uploadConanExport)
+    createRoute(builder, uploadConanExportZip(), AssetKind.CONAN_EXPORT, hostedHandler.uploadConanExport)
```

Comment:
```suggestion
    createRoute(builder, uploadConanExportZip(), CONAN_EXPORT, hostedHandler.uploadConanExport)
```

SUGGESTION: We should likely static import this, as well as the CONAN_SOURCES above it (don't need to do that in this PR, but it's inconsistent)

Reply:
CONAN_SOURCES and DOWNLOAD_URL are referenced by AssedKind due to name conflict with class member strings.